### DISCUSSION TITLE
Assigning tags via drag & drop

### DISCUSSION THREAD

**Discussion by tobias-klein** [ezra-bible-app/ezra-bible-app#598] [Ideas]
I was just assigning some tags using the Android app and I was then thinking: How about supporting a drag and drop operation for assigning tags? That would be even faster then two clicks/taps, because you wouldn't need to select a verse before assigning the tag. You could just drop it on the verse!

@augustin-colesnic @zhuiks @debritto

As usual I am interested in your feedback.

**Comment by debritto** [comment]
Awesome idea! Very functional and intuitive. ;)

**Comment by zhuiks** [comment]
I think it's a great idea. But to be more clear from UX perspective, the tags in the tag panel should look similar to the Bible browser / verse list window (colored label chips). Maybe it can be implemented with separate drag zones under tag groups #540 (each group can have their own drag zone).
